windows/src/desktop/kmshell/install/UfrmInstallKeyboard.pas

```diff
@@ -274,6 +274,7 @@ function InstallFile(Owner: TComponent; const FileName: string; ASilent, ANoWelc
   try
     Silent := ASilent;
     InstallFile := FileName;
+//TODO:    DefaultBCP47 := BCP47;
```

Comment:
> This does not yet implement the keyboard language selection, although the BCP 47 code is passed into the InstallFile function ready for that.

I see there's also no default value provided for cases where `''` is passed in for `BCP47`?

Reply:
> I see there's also no default value provided for cases where `''` is passed in for `BCP47`?

The default value will be extracted from the package. But I have to do that code (it's similar to the `InstallKeyboards` code which already does this.